docs/designs/output.md

```diff
@@ -79,44 +103,75 @@ Different files can share the same `{site-url}` or `{site-path}` due to versioni
+- Content for static rendering using ugly URL(OutputType: `Html`)
 
     | | |
     |------ |----|
-    | `url` | https://docs.microsoft.com/en-us/dotnet/api/system.string.html |
-    | `site-url` | /dotnet/api/system.string.html |
+    | `url` | https://self-hosting/en-us/dotnet/01ddf122/api/system.string.html |
+    | `site-url` | /dotnet/01ddf122/api/system.string.html |
     | `base-path` | dotnet |
-    | `site-path` | api/system.string.html |
-    | `output-path` | dotnet/api/system.string.html |
+    | `site-path` | 01ddf122/api/system.string.html |
+    | `output-path` | dotnet/01ddf122/api/system.string.html |
 
-- Table of Contents
+- Table of Contents in Docs hosting(OutputType: `Json`):
 
     | | |
     |------ |----|
-    | `url` | https://docs.microsoft.com/en-us/dotnet/api/TOC.json |
+    | `url` | https://docs.microsoft.com/en-us/dotnet/api/TOC.json?view=netstandard-2.0 |
     | `site-url` | /dotnet/api/TOC.json |
     | `base-path` | dotnet |
     | `site-path` | api/TOC.json |
-    | `output-path` | dotnet/api/TOC.json |
+    | `output-path` | dotnet/01ddf122/api/TOC.json |
+
+- Table of Contents in static hosting using pretty URL(OutputType: `Html`):
+
+    | | |
+    |------ |----|
+    | `url` | https://docs.microsoft.com/en-us/dotnet/01ddf122/api/TOC/ |
+    | `site-url` | /dotnet/01ddf122/api/TOC/ |
+    | `base-path` | dotnet |
+    | `site-path` | 01ddf122/api/TOC/index.html |
+    | `output-path` | dotnet/01ddf122/api/TOC/index.html |
 
-- Image
+- Table of Contents in static hosting using ugly URL(OutputType: `Html`):
 
     | | |
     |------ |----|
-    | `url` | https://docs.microsoft.com/en-us/dotnet/api/thumbnail.png |
+    | `url` | https://docs.microsoft.com/en-us/dotnet/01ddf122/api/TOC.html |
+    | `site-url` | /dotnet/01ddf122/api/TOC.html |
+    | `base-path` | dotnet |
+    | `site-path` | 01ddf122/api/TOC.html |
+    | `output-path` | dotnet/01ddf122/api/TOC.html |
+
+- Image in Docs hosting:
+
+    | | |
+    |------ |----|
+    | `url` | https://docs.microsoft.com/en-us/dotnet/api/thumbnail.png?view=netstandard-2.0 |
     | `site-url` | /dotnet/api/thumbnail.png |
     | `site-path` | dotnet/api/thumbnail.png |
-    | `output-path` | dotnet/api/thumbnail.png |
+    | `output-path` | dotnet/01ddf122/api/thumbnail.png |
+
+- Image in static hosting:
+
+    | | |
+    |------ |----|
+    | `url` | https://docs.microsoft.com/en-us/dotnet/01ddf122/api/thumbnail.png |
+    | `site-url` | /dotnet/01ddf122/api/thumbnail.png |
+    | `site-path` | dotnet/01ddf122/api/thumbnail.png |
+    | `output-path` | dotnet/01ddf122/api/thumbnail.png |
+
+> Note: For static page versioning, the ideal output path should use the human readable `moniker` instead of monikerHash, but that require coping the same content to different output folder of each version. For the time been, the static output is only for generating PDF, the URL does not appear in PDF now, so we can still use the monikerHash.
```

Comment:
```suggestion
> Note: For static page versioning, the ideal output path should use the human readable `moniker` instead of monikerHash, but that require coping the same content to different output folder of each version. For the time being, the static output is only for generating PDF, the URL does not appear in PDF now, so we can still use the monikerHash.
```